buildpack.md

```diff
@@ -857,6 +858,9 @@ For each process, the buildpack:
   - A path to an executable or the file name of an executable in `$PATH`, if `args` is a list with zero or more elements.
 - MAY specify an `args` list to be passed directly to the specified executable.
 - MAY specify a `direct` boolean that bypasses the shell.
+- MAY specify a `default` boolean that indicates that the process type should be selected as the [buildpack-provided default](https://github.com/buildpacks/spec/blob/main/platform.md#outputs-4) during the export phase.
+
+An individual buildpack may only specify one process type with `default = true`. If multiple buildpacks define processes of the same type, the lifecycle MUST use the last process type definition ordered by buildpack execution for the combined process list. The lifecycle MUST select the last process type with `default = true` as the buildpack-provided default.
```

Comment:
I think it's worth adding an explanation of the two following edge cases:
* New buildpack with a default worker process → new buildpack with a non-default worker process
⇒ no default + override warning
* New buildpack with a default Y process → new buildpack with a default X process → new buildpack with a non-default Y process
⇒ default = X (no warning)

Reply:
I think the first bullet is covered in `the lifecycle MUST use the last process type definition ordered by buildpack execution for the combined process list`

Reply:
I think the second bullet is covered in `The lifecycle MUST select the last process type with `default = true`

I'm not opposed to making it more explicit, but in general the spec reads pretty tersely (for better or worse) so maybe it's more consistent this way. I'm happy to do whatever we think is the right thing.

Reply:
I'm fine with not describing the second bullet because as you wrote, it's probably too much for the spec.
But I think it's worth making the first bullet clearer, meaning the fact that a non-default process can override a default process.

Reply:
That's fair. I expanded on it a bit - lmk what you think.

Reply:
Updated in https://github.com/buildpacks/spec/pull/205/commits/4356d833be2d1ebf7751db87a19fd4317ec42b9d